docs/source/development-guide/style-guide/naming-conventions.rst

```diff
@@ -21,7 +21,7 @@ The general filename convention is as follows::
   * CoDICE: ``l0``, ``l1a``, ``l1b``, ``l2``, ``l3``
   * GLOWS: ``l0``, ``l1a``, ``l1b``, ``l2``, ``l3a``, ``l3b``, ``l3c``, ``l3d``
   * HIT: ``l0``, ``l1a``, ``l1b``, ``l2``, ``l3``
-  * IDEX: ``l0``, ``l1``, ``l2``, ``l3``
+  * IDEX: ``l0``, ``l1a``, ``l1b``, ``l2``, ``l3``
```

Comment:
Do we need to update the `sds-data-manager` repo with this as well for the dependency graph? If so, can you open an issue for that.

Reply:
Ah yes, good call.